ERCS/erc-7730.md

```diff
@@ -0,0 +1,440 @@
+---
+eip: 7730
+title: Structured Data Clear Signing Format
+description: Draft specification of a json format providing additional description of how to clear-sign smartcontract calls and EIP712 messages on any wallets
+author: Laurent Castillo (@lcastillo-ledger)
+discussions-to: https://github.com/LedgerHQ/clear-signing-format-spec/issues
+status: Draft
+type: Standards Track
+category: Interface
+created: 2024-02-07
+requires: 712
+---
+
+## Abstract
+
+This specification defines a JSON format carrying additional information required to correctly display structured data to a human for review on a wallet screen, before signature by the wallet. In the current scope, structured data supported are smart contract calls in EVM transactions and EIP 712 structured messages. 
```

Comment:
It would be nice to include a summary of how the proposal works. For example, what additional information is carried?

After reading your Abstract, your audience should have a high-level (but still technical) idea of what your proposal does.